bf2fc6cc711aee1a0c2a/kas-fleet-manager
# Added spectral file, modified Make file, removed pkg/api/naming_stand…

<-- Please use the PR template and provide as much relevant information as you can, otherwise your PR may not get reviewed. -->

## Description

   Adding  Spectral linting for our OpenAPI docs. https://issues.redhat.com/browse/MGDSTRM-4410

## Verification Steps

   Make file has been updated to install spectral and rhoas/spectral-ruleset. 

   specintall and validateOpenapiSpec commands are added to test the functionality

   Run the Make command validateOpenapiSpec to check the functionality


## Checklist (Definition of Done)
